app/storybook/documentation/ui-kit/components/PieChart.mdx

```diff
@@ -0,0 +1,51 @@
+import { ArgTypes, Meta } from '@storybook/blocks'
+import LinkTo from '@storybook/addon-links/react'
+import { prettier } from '../../../src/utils'
+import { SourceCode } from '../../../.storybook/blocks/SourceCode'
+import * as PieChartStories from '../../../../../packages/ui-kit/src/components/PieChart/PieChart.stories'
+import * as PieChartQueryPropsStories from '../../../../../packages/ui-kit/src/components/PieChart/PieChartQueryProps.stories'
+import * as PieChartPropsStories from '../../../../../packages/ui-kit/src/components/PieChart/PieChartProps.stories'
+
+<Meta title="Components/PieChart" of={PieChartStories} />
+
+# PieChart overview
+
+PieChart component is designed to create Propel PieChart metric visualizations for your front-end
+applications. With two props in Static mode, `headers` and `rows`, the PieChart component provides a simple way to
+display your data.
```

Comment:
Not 100% sure about the copy here. I revised it to be a little more similar to the Counter overview; however, I also noticed that in the Counter overview, we link to studio.apollographql.com, which does not load unless logged in. We need to replace such URLs. I've opened https://github.com/propeldata/ui-kit/pull/154 for that.

```suggestion
The PieChart component is designed to create pie chart (AKA doughnut chart) visualizations for your front-end
applications. With two props in Static mode, `headers` and `rows`, the PieChart component provides a simple way to
display your data. If you
opt for the Connected mode, the component will accept the `query` prop, which loads the data automatically using Propel's GraphQL API.
```